### PR TITLE
Add configurable callback URL to 3-legged auth

### PR DESCRIPTION
**Pull request by gadhagod**
You can now set a callback URL with the `request_authorization` function by setting the parameter `callback_url`.

### CODE REVIEW

File: schoolopy/authentication.py [hunk 1]
```diff
@@ -56,7 +56,9 @@ def _request_header(self):
             'Content-Type': 'application/json'
         }
 
-    def request_authorization(self):
+    def request_authorization(self, callback_url=None):
+        if callback_url == None:
+            callback_url = self.DOMAIN_ROOT
```
Comment (ErikBoesen) [inline schoolopy/authentication.py:61]: Why not just include this in the function param? `...(self, callback_url=DOMAIN_ROOT):`

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

**Comment by ErikBoesen** [comment]
Also, wow, eighth grade? Keep it up man, you're gonna do great things.

File: schoolopy/authentication.py (hunk 1 above)
Comment (ErikBoesen) [reply schoolopy/authentication.py:61]: Update: forgot that's dynamically set, this looks good

**Review by ErikBoesen — APPROVED** [review]

**Comment by gadhagod** [comment]
Thanks! Congrats on becoming a SWE at schoology.

**Pull request merged**